### PR TITLE
fix(authentication): autologin link corrections

### PR DESCRIPTION
**Pull request by EdytaSki**

### CODE REVIEW

**Comment by lpinsivy** [comment]
Please provide a correct title for your PR:

fix(authentication): autologin link corrections

File: www/front_src/src/components/userMenu/index.js [hunk 1]
```diff
@@ -74,20 +75,19 @@ class UserMenu extends Component {
...
-  getAutologinLink = () => {
-    const { username, autologinkey } = this.props.data
-
-    return window.location.href + '?autologin=1&useralias=' + username + '&token=' + autologinkey
-  };
-
   render() {
     const { data, toggled, copied } = this.state;
-
+
```
Comment (kduret) [inline www/front_src/src/components/userMenu/index.js]: remove those spaces

File: www/front_src/src/components/userMenu/index.js [hunk 2]
```diff
@@ -74,20 +75,19 @@ class UserMenu extends Component {
...
-
+  
     if (!data) {
       return null;
     }
     const { fullname, username, autologinkey } = data;
 
+    //creating autologin link, getting href, testing if there is a parameter, then generating link : if '?' then &autologin(etc.)
+    const gethref = window.location.href,
+          conditionnedhref = window.location.search ? gethref  + '&': gethref + '?',
```
Comment (kduret) [inline www/front_src/src/components/userMenu/index.js]: conditionedhref = gethref + (window.location.search ? '&' : '?'),

File: www/front_src/src/components/userMenu/index.js [hunk 3]
```diff
@@ -88,6 +83,11 @@ class UserMenu extends Component {
     }
     const { fullname, username, autologinkey } = data;
 
+    //creating autologin link, getting href, testing if there is a parameter, then generating link : if '?' then &autologin(etc.)
+    const gethref = window.location.href,
+          conditionnedhref = gethref + (window.location.search ?'&':'?'),
```
Comment (sc979) [inline www/front_src/src/components/userMenu/index.js]: please add space before and after the ternary "?" and ":"

**Pull request merged**